packages/plugins/script/Sample.s.sol

```diff
@@ -11,37 +11,52 @@ contract Sample is Sphinx {
     MyContract1 myContract;
 
     function setUp() public {
-        sphinxConfig.projectName = "test_project";
+        sphinxConfig.projectName = "test_project_2";
         sphinxConfig.owners = [0x9fd58Bf0F2E6125Ffb0CBFa9AE91893Dbc1D5c51];
         sphinxConfig.threshold = 1;
-        sphinxConfig.testnets = [Network.sepolia, Network.arbitrum_sepolia];
+        sphinxConfig.testnets = [
+            Network.sepolia,
+            Network.arbitrum_sepolia,
+            Network.optimism_sepolia,
+            Network.avalanche_fuji,
+            Network.polygon_mumbai,
+            Network.fantom_testnet,
+            Network.bnb_testnet,
+            Network.gnosis_chiado,
+            Network.linea_goerli,
+            Network.polygon_zkevm_goerli,
+            Network.base_sepolia,
+            Network.celo_alfajores,
+            // Network.moonbase_alpha,
+            Network.evmos_testnet,
+            Network.kava_testnet,
+            Network.scroll_sepolia,
+            Network.zora_sepolia,
+            Network.rari_sepolia
+        ];
         sphinxConfig.mainnets = [Network.ethereum, Network.arbitrum];
         sphinxConfig.orgId = "clo6byksj0001cbld6lelntej";
-        sphinxConfig.saltNonce = 0;
+        sphinxConfig.saltNonce = 2112414;
```

Comment:
I mean changes to this file don't really matter since it's only used for testing. At least, I only use it for testing. I can revert the changes though, idc